src/assets/scss/components/_login.scss

```diff
@@ -0,0 +1,12 @@
+.login-form {
+  text-align: center;
+  input {
+    margin-bottom: 10px;
+  }
+  button {
+    width: 100%;
+    &.primary {
+      margin-bottom: 10px;
+    }
+  }
```

Comment:
Do we really need this, or can we make it generic.

Reply:
Removed, this is not needed